python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/port-channel-interfaces.j2

```diff
@@ -124,14 +148,29 @@
 
 ##### VLAN Translations
 
-| Interface | From VLAN ID(s) | To VLAN ID | Direction |
-| --------- | --------------- | -----------| --------- |
+| Interface |  Direction | From VLAN ID(s) | To VLAN ID | From Inner VLAN ID | To Inner VLAN ID | Network | Dot1q-tunnel |
+| --------- |  --------- | --------------- | ---------- | ------------------ | ---------------- | ------- | ------------ |
 {%         for port_channel_interface in port_channel_interfaces | arista.avd.natural_sort('name') %}
-{%             if port_channel_interface.vlan_translations is arista.avd.defined %}
+{%             if port_channel_interface.switchport.vlan_translations is arista.avd.defined %}
+{%                 for vlan_translation in port_channel_interface.switchport.vlan_translations.direction_both | arista.avd.natural_sort('from') %}
+| {{ port_channel_interface.name }} | both | {{ vlan_translation.from }} | {{ vlan_translation.to | arista.avd.default('-') }} | {{ vlan_translation.inner_vlan_from | arista.avd.default('-') }} | - | {{ vlan_translation.network | arista.avd.default('-') }} | {{ vlan_translation.dot1q_tunnel | arista.avd.default('-') }} |
```

Comment:
Do we really need arista.avd.default('-') for To VLAN ID as this is required key

Reply:
fixed